libs/platform/src/lib/form/multi-combobox/multi-combobox/multi-combobox.component.html

```diff
@@ -247,11 +247,7 @@
 
 <ng-template let-optionItem="optionItem" let-index="index" #optionItemSource>
     <ng-container *ngIf="!optionItemTemplate">
-        <span
-            fd-list-title
-            [attr.title]="optionItem.label"
-            [innerHTML]="optionItem.label | highlight: inputText"
-        ></span>
+        <span fd-list-title [attr.title]="optionItem.label">{{ optionItem.label }}</span>
```

Comment:
The root of the issue actually in the `innerHTML`, because when we use `<` it's perceived as tag, not text. As you changed it to interpolation `{{ ... }}` it's safe to keep `| highlight: inputText` pipe.

Reply:
Could also use innerText I think

Reply:
Anyways I put the highlight pipe back

Reply:
@platon-rov adding **highlight: inputText** in string interpolation is actually  creating different issue  while searching in multi-combobox input area as shown in the picture.
![image](https://user-images.githubusercontent.com/110972289/189191296-164c7728-603b-4ae2-9b12-3a0d9d327255.png)

However I re-worked and found a different solution using [innerHTML] but replace "<" with unicode which works fine in my local side by side **highlight: inputText** .
![image](https://user-images.githubusercontent.com/110972289/189192134-80cc8cfb-a373-41e3-a214-77f16843ede9.png)

please let me know if that is okay.

Reply:
Interpolation is not cooperating with the highlight pipe:
![Screen Shot 2022-09-08 at 12 05 41 PM](https://user-images.githubusercontent.com/2471874/189194353-6956e5f4-d1dc-48a5-875a-7523d7701df3.png)

Interesting that the span is being parsed though, investigating now